src/main/java/seedu/findvisor/logic/commands/DeleteTagCommand.java

```diff
@@ -0,0 +1,146 @@
+package seedu.findvisor.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.findvisor.logic.parser.CliSyntax.PREFIX_TAG;
+import static seedu.findvisor.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+import seedu.findvisor.commons.core.index.Index;
+import seedu.findvisor.commons.util.ToStringBuilder;
+import seedu.findvisor.logic.Messages;
+import seedu.findvisor.logic.commands.exceptions.CommandException;
+import seedu.findvisor.model.Model;
+import seedu.findvisor.model.person.Address;
+import seedu.findvisor.model.person.Email;
+import seedu.findvisor.model.person.Meeting;
+import seedu.findvisor.model.person.Name;
+import seedu.findvisor.model.person.Person;
+import seedu.findvisor.model.person.Phone;
+import seedu.findvisor.model.person.Remark;
+import seedu.findvisor.model.tag.Tag;
+
+/**
+ * Deletes an existing tag of a person identified using it's displayed index
+ * from the
+ * address book.
+ */
+public class DeleteTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "deletetag";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the tag associated with a particular person "
+            + "identified by the index number used in the displayed person list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "[" + PREFIX_TAG + "TAG]\n"
+            + "Example: " + COMMAND_WORD + " 1 t/tag";
```

Comment:
Perhaps would be better to change the example tag to something more descriptive?
```suggestion
            + "Example: " + COMMAND_WORD + " 1 t/PRUTravellerProtect";
```